client.go

```diff
@@ -100,68 +89,45 @@ func (c *Client) SetMaxRetries(maxRetries int) {
 // Do calls ctxhttp.Do with the addition of exponential backoff
 func (c *Client) Do(ctx context.Context, req *http.Request) (*http.Response, error) {
 
-	if len(c.AuthToken) > 0 {
-		// only add this header if not already set (e.g. for authClient)
-		if len(req.Header.Get(common.AuthHeaderKey)) == 0 {
-			common.AddServiceTokenHeader(req, c.AuthToken)
-		}
-	}
-	if len(c.DownloadServiceToken) > 0 {
-		// only add this header if not already set
-		if len(req.Header.Get(common.DownloadServiceHeaderKey)) == 0 {
-			common.AddDownloadServiceTokenHeader(req, c.DownloadServiceToken)
-		}
-	}
-	if common.IsUserPresent(ctx) {
-		// only add this header if not already set
-		if len(req.Header.Get(common.UserHeaderKey)) == 0 {
-			common.AddUserHeader(req, common.User(ctx))
-		}
-	}
-	if common.IsFlorenceIdentityPresent(ctx) {
-		common.SetFlorenceHeader(ctx, req)
-	}
-
 	// get any existing correlation-id (might be "id1,id2"), append a new one, add to headers
 	upstreamCorrelationIds := common.GetRequestId(ctx)
```

Comment:
```suggestion
	upstreamCorrelationIDs := common.GetRequestId(ctx)
```
if you want to be consistent about `Id`->`ID`

typ.